CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 * [#5738](https://github.com/spacemeshos/go-spacemesh/pull/5738) sql: fix epoch ATX ID cache deadlock
 
+* [#5745](https://github.com/spacemeshos/go-spacemesh/pull/5745) fetch: fix handling hash request errors in
```

Comment:
i am not sure if this requires a changelog, i doubt that somebody started to use streaming beside us